### DISCUSSION TITLE
EON switching on TA or special PTY

### DISCUSSION THREAD

**Discussion by lxsxl** [PE5PVB/TEF6686_ESP32#411] [Ideas]
For future developement you could implement two kind of EON switching:
1. If TA-bit is set high for an EON-network (which also carries TP-bit), then tune to this network (if selected to do so, needs a new menu item); also look for group 14B for fast switching information.
2. If EON-PTY transmits 31 (Alarm), then tune to this network (if selected to do so, needs a new menu item); could possibly also be done for 30 (Alarm Test).

**Comment by PE5PVB** [comment]
Ok so you mean when a traffic announcement is going on at another service within the EON switch to that service? What is the difference between group 14A and group 14B? Can you get/give me an example?

**Reply by lxsxl** [reply]
Yes, that's what I mean. Difference between 14A and 14B is that a complete group 14A cycle may last a minute or even longer, so if 14A hits the announcement-service the announcement may already have ended. Group 14B however will be sent once when announcement starts (and maybe also when it ends), so you have a fast switching information. I will try to get a record for you.

**Comment by lxsxl** [comment]
***@***.*** sent you MPX record group 14B

1 item, 71.1 MB in total ・ Expires on 22 February, 2024


Download link:
https://wetransfer.com/downloads/8ab163327bc7ff7e74838039802ba6bc20240215211827/95f74610a0f2bcf25919cfbe3bf3021020240215211836/673570?trk=TRN_TDL_01&utm_campaign=TRN_TDL_01&utm_medium=email&utm_source=sendgrid

**Comment by lxsxl** [comment]
14B groups are sent at about 2 min and again some seconds later
